libmscore/midimapping.cpp

```diff
@@ -137,7 +137,9 @@ int MasterScore::getNextFreeDrumMidiMapping()
 
 void MasterScore::rebuildExcerptsMidiMapping()
 {
-    for (Excerpt* ex : excerpts()) {
+    auto allExcerpts = excerpts();
+    allExcerpts.append(albumExcerpts());
```

Comment:
I don't really know how efficient it is to make a copy of `excerpts` and append another load of excerpts to it, but since it's only pointers maybe it's fine.